docs/develop/sdks-integrations/aws-secrets-manager.md

```diff
@@ -15,7 +15,7 @@ It's best practice to securely store your Momento authentication token. If you a
 
 :::info
 
-Just as you should reuse your Momento `CacheClient` and `TopicClient` objects when possible, so should you with the Momento auth token from AWS Secrets Manager. Otherwise you risk adding cost, both in time and money, to each Momento API call for the round trip to AWS Secrets Manager.
+Just as you should reuse your Momento `CacheClient` and `TopicClient` objects when possible, so should you with the Momento auth token from AWS Secrets Manager. Otherwise you risk adding cost, both in time and money, to each Momento API call for the round trip to AWS Secrets Manager. For high throughput use cases, you might also encounter throttling from AWS Secrets Manager - to scale beyond this, it's best to use client side caching of the credentials as detailed in this [AWS blog](https://aws.amazon.com/blogs/security/use-aws-secrets-manager-client-side-caching-libraries-to-improve-the-availability-and-latency-of-using-your-secrets/).
```

Comment:
Maybe instead of `high throughput`  we say like workloads w/ high number of concurrent containers or spiky lamba workloads or something along those lines?

Reply:
I agree with Ellery. I like the more specific examples for this. "high throughput" is relative to the person. Even "high number" is too. So the more specific we can be, the better, IMO.

Reply:
How about "high concurrency"? I'm not even sure that's correct though. Someone could mistakenly set up a loop in a single client that attempts to GetSecretValue for every cache read. GetSecretValue is limited to 10k rps (per account/region) - hard limit - so not all that hard to exceed it.

Reply:
@eaddingtonwhite @NoSQLKnowHow

Reply:
The more I read this, the more I ask why define this at all? Even at small scale, if you are getting the auth token from AWS SM too often, you are wasting operation time and hard money in each call.

Just say "To avoid the added cost and potential throttling, it's best to use client-side caching of the auth token...."

Reply:
That's a good call Kirk - will adjust and get a new version out for review.

Reply:
Okay @NoSQLKnowHow and @eaddingtonwhite - take a look at the updated wording and see what you think.

Reply:
LGTM